<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elpida::Application::MemoryBenchmarkView</class>
 <widget class="QWidget" name="MemoryBenchmarkView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="gbBenchmarkConfiguration">
     <property name="title">
      <string>Benchmark Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="lblMultiplier">
        <property name="text">
         <string>Multiplier:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spnMultiplier">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>1.250000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblStartSize">
        <property name="text">
         <string>Start size:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spnIterations">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblIterations">
        <property name="text">
         <string>Iterations:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spnStartSize">
        <property name="minimum">
         <number>64</number>
        </property>
        <property name="maximum">
         <number>2100000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbExecution">
     <property name="title">
      <string>Execution</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
        <item>
         <widget class="QLabel" name="lblCurrentBenchmark">
          <property name="text">
           <string>Current benchmark:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCurrentBenchmarkValue">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QProgressBar" name="pbProgress"/>
        </item>
        <item>
         <widget class="QLabel" name="lblProgress">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0/0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="bpStart">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatus">
          <property name="text">
           <string>Ready</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbBenchmark">
     <property name="title">
      <string>Benchmark</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbMemoryLatency">
        <property name="text">
         <string>Latency</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbMemoryReadBandwidth">
        <property name="text">
         <string>Read bandwidth</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="gbMemoryBenchmarkResults">
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="twMemoryBenchmarkResults">
        <column>
         <property name="text">
          <string>Size</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Score</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
